biofefi/components/synthetic_data.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+from sklearn.datasets import make_classification, make_regression
+
+from biofefi.options.enums import ProblemTypes
+from biofefi.options.synthetic_data_opts import SyntheticDataOptions
+from biofefi.utils.assertion import DataLoaderChecker
+
+
+class CreateSyntheticData:
```

Comment:
```suggestion
class SyntheticDataBuilder:
```
Class names should be nouns

Reply:
Ok Dan